test/test.js

```diff
@@ -70,6 +70,21 @@ test('linting of incorrect Markdown file fails', async t => {
   }
 });
 
+test('linting of incorrect Markdown file fails prints issues as json', async t => {
+  try {
+    await execa('../markdownlint.js',
+      ['--config', 'test-config.json', 'incorrect.md', '--json'],
+      {stripFinalNewline: false});
+    t.fail();
+  } catch (error) {
+    t.is(error.stdout, '');
+    const issues = JSON.parse(error.stderr);
+    t.is(issues.length, 8);
+    t.is(issues[0].fileName, "incorrect.md");
```

Comment:
Could you please validate the complete structure of one object so it won't change unexpectedly?

Reply:
Sure.